### DISCUSSION TITLE
Water Credit Asset Class into Registry.

### DISCUSSION THREAD

**Discussion by Cloudunetworks** [regen-network/regen-ledger#862] [Ideas]
How can we use the idea of carbon credit baskets in a way that applies to water.
 - We need methodologies for water sourcing, quality, and management.
 -  Sourcing may come from wells, rainwater holding tanks, or natural sources from land. ( this may depend on bio region )
    + This may be a step into multiple credit registries
- Identifying the quality of water?
    + Need help getting info. We need a list of projects dedicated to clean water.
- Management refers to the usage of water. This will depend on the source of water. We should have discussions around how to implement carbon negative approaches and governing the water allocation design.

Please fill free to add anything to help with this initiative. As the Carbon Market starts to take rise. We can companion the water conservation to it.

**Comment by ryanchristo** [comment]
This would be great to post in our Discord (https://discord.gg/regen-network). We use discussions within this repository for the regen-ledger codebase and only our engineers are receiving notifications. Another option would be our forum (https://commonwealth.im/regen/), which we recently migrated from Discourse to Commonwealth and are in the process of making it more user friendly, but I would start with Discord.
